framework/src/main/java/org/checkerframework/common/accumulation/AccumulationAnnotatedTypeFactory.java

```diff
@@ -248,6 +320,21 @@ public QualifierHierarchy createQualifierHierarchy(MultiGraphFactory factory) {
      *        |
      *      bottom
      * </pre>
+     *
+     * Predicate subtyping is defined as follows:
```

Comment:
It might be good to document what the "right thing" to do would be, and why we deviate from that.  I think the "right thing" here is logical implication, i.e., P is a subtype of Q iff P implies Q.  But we don't bake in the implication check to keep things simple and minimize dependencies.

Reply:
I'll add a comment.